src/lib/workspace/AllAnalyses.tsx

```diff
@@ -336,20 +336,18 @@ export function AllAnalyses(props: Props) {
         },
         {
           element: (
-            <FormControlLabel
-              control={
-                <Switch
-                  color="primary"
-                  size="small"
-                  checked={sortPinned}
-                  onChange={(e) => setSortPinned(e.target.checked)}
-                />
-              }
+            <Toggle
               label="Sort pinned to top"
-              style={{
-                padding: '1em',
-              }}
+              labelPosition="right"
+              value={sortPinned}
+              onChange={setSortPinned}
               disabled={pinnedAnalyses.length === 0}
+              styleOverrides={{
+                container: { marginLeft: '1em' },
+                mainColor: { hue: red, level: 500 },
```

Comment:
why is this red instead of the theme color?

Reply:
And about the red---whoops, that was just me testing that prop. Forgot to replace it. Good catch!